examples/src/main/java/com/basistech/rosette/examples/TopicsExample.java

```diff
@@ -31,7 +31,7 @@ public static void main(String[] args) {
     }
 
     private void run() throws Exception {
-        String sampleData = "Original Ghostbuster Dan Aykroyd, who also co-wrote the 1984 Ghostbusters film, couldn’t be more pleased with the new all-female Ghostbusters cast, telling The Hollywood Reporter, “The Aykroyd family is delighted by this inheritance of the Ghostbusters torch by these most magnificent women in comedy.";
+        String sampleData = "Lily Collins is in talks to join Nicholas Hoult in Chernin Entertainment and Fox Searchlight's J.R.R. Tolkien biopic Tolkien. Anthony Boyle, known for playing Scorpius Malfoy in the British play Harry Potter and the Cursed Child, also has signed on for the film centered on the famed author. In Tolkien, Hoult will play the author of the Hobbit and Lord of the Rings book series that were later adapted into two Hollywood trilogies from Peter Jackson. Dome Karukoski is directing the project.";
```

Comment:
Should **sampleData** be **topicsData** in order to match with the rosette-api-example-data?

Reply:
good catch, pushed the update